packages/web-components/src/components/ic-date-input/ic-date-input.stories.mdx

```diff
@@ -269,12 +328,42 @@ import readme from "./readme.md";
   <Story name="Dates from now disabled" parameters={{ loki: { skip: true } }}>
     {html`<ic-date-input
         label="When would you like to collect your coffee?"
-        disabled-dates="from-now"
+        disable-from-now
       ></ic-date-input>
       <ic-typography>Enter a date in the future</ic-typography> `}
   </Story>
 </Canvas>
 
+### Disabled days
+
+<Canvas>
+  <Story name="Disabled days" parameters={{ loki: { skip: true } }}>
+    {html`<ic-date-input
+        label="When would you like to collect your coffee?"
+      ></ic-date-input>
+      <script>
+        var dateInput = document.querySelector("ic-date-input");
+        dateInput.disableDays = [0, 1];
```

Comment:
Assuming these are coming from the date picker, would it make more sense if the `disableDays` were an array of actual days rather than indexes and we convert them to indexes internally?
Or....could accept both?

Reply:
Yeah I'm not sure, I presume there was a reason @ad9242 implemented it like this...

Reply:
Let's park this for now. Assuming this is being passed down from `Date Picker` so maybe this question is more related to that component.